resume.md

```diff
@@ -1,49 +1,48 @@
-# FIRST LAST NAME
+# Daniel Qian
 
-(123) 456 - 7890 | email@gmail.com
+(925) 922 - 4281 | danielq3@illinois.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+github.com/DAQIANN | linkedin.com/danielq5/
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**University of Illinois Urbana Champaign** *May 2023* *GPA 3.89* *Dean's List*
+* Bachelor of Science in Mathematics and Computer Science 
+* Minor in Statistics 
+* **Coursework**: Data Structures, Numerical Methods, Software Design Studio, Discrete Structures, Linear Algebra, Statistical Analysis, Statistics and Probability
+
+## EXPERIENCE
+**Course Assistant | UIUC | Champaign, IL  |September 2020 – Present**
+* Founding member of CS 128 (Introduction to Computer Science II), a new core Computer Science class to be implemented Spring 2021
+* Developing engaging assignments for future students surrounding C++ topics
+* Working cohesively with other assistants to produce the course material
+
+**Sandia National Laboratories Space Blimp Challenge at Illinois | November 2019**
+* Led team that developed new software that improves the management of experimental flight data 
+* Effectively utilized Java to parse through given data for maximum time efficiency within Hackerrank prompt
+
+**Alameda County Science and Engineering Fair | April 2018**
+* Presented a research study focused on the accuracy of certain Artificial Intelligence technologies and Machine Learning to industry professionals
+* Analyzed Artificial Intelligence services such as Amazon Rekognition Services, Google Cloud Vision Service, Microsoft’s Cognitive Services Vision and Face on their image correctness, accuracy, and comprehensiveness 
+
+## LEADERSHIP/ACTIVITIES
+**Illinois Formula Electric | August 2020 - Present**
+* Member of the Data Acquisition Analysis (DAQA) subsystem
+* Worked collaboratively with clubmates to enhance data visualization with Python by implementing Pandas and Pymongo
```

Comment:
More about this since this is technical ^ be specific on how you used Pandas and Pymongo, why didnt other technologies work? what is so cool about these frameworks?  Drop some techincal language here